### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by Guy-Gustave**
the calculator pull-request for the second milestone. to display unfunctional calculator

### CODE REVIEW

**Review by krys2fa — CHANGES_REQUESTED** [review]
# Status: Marked as Invalid! ⚠️  
Please note that your Code Review Request is being marked as invalid. 
This is because your PR has linters failing. Please ensure your PR has all checks passing and then request for another review.  

Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: '',
+};
```
Comment (krys2fa) [inline src/components/Display.js]: - [ ] the result prop is expected to be a String with default value = 0 as stated in ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)).
**Hint:**
```
Display.defaultProps = {
  value: "0"
};

```

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome work @Guy-Gustave 👏 
Your project directory structure is coming along nicely and your code base is clean. 💪 
Just one more improvement to make to complete this milestone.
### Status: Changes Required ♻️ 

- [ ] Kindly check the comments for the improvement to make.

Please request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by Guy-Gustave** [comment]
I am grateful for the review because you helped me realize well the requirements that I had to follow. all of them I fixed. thank you.

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi @Guy-Gustave :wave: 

Go on with your next task.

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Inyama Henry ```

**Pull request merged**